src/templates/en/2019/accessibility_statement.html

```diff
@@ -0,0 +1,203 @@
+{% extends "en/2019/base.html" %}
+
+{% block title %}Accessibility Statement | The Web Almanac by HTTP Archive{% endblock %}
+
+{% block description %}Describes the accessibilty goals of the HTTP Archive Web Almanac{% endblock %}
+
+{% block meta %}
```

Comment:
Do we need to redeclare all of this metadata or is the default sufficient?

Reply:
We don't have a default one, except a very light one in src/templates/base.html so I say we need this in line with all the other pages.

Reply:
We should try to tackle this as part of #673 . Resolving this for now.